src/front/wgsl.rs

```diff
@@ -340,6 +338,7 @@ pub struct ParseError<'a> {
     pub pos: (usize, usize),
 }
 
+#[derive(Debug, Default)]
```

Comment:
why do we need default here?

Reply:
So this is clippy's new_without_default lint, while there is a new method that takes no args I don't think there's any harm in having a Default impl, and it does make some things slightly nicer.

Reply:
hm, I'm not certain that this is a good idea. `Default` should be used for things that are truly "default", not just *any* thing with this format of constructor. I.e. the arguments of a constructor don't imply a specific semantics that `Default` does.
Can we have that change reverted and the `new_without_default` lint disabled?

Reply:
Yeah, I've always been slightly dubious of that lint myself.